### ISSUE TITLE
Cannot use versions higher than 0.9.78

### ISSUE THREAD

**Issue by jmatos** [Hammerspoon/hammerspoon#2840]
OS: MacOS Big Sur V. 11.1

I see the below error in the Hammerspoon console when I was using the latest version 0.9.87 - so I switched back to 0.9.78.

I was trying to use the windows bindings - CTRL-S and "h" or "k", etc..., but only CTRL-S <enter> would work - switching back to 0.9.78 restored the windows bindings. 

--- ERROR ---
2021-04-21 17:42:55: 17:42:55 ERROR:   LuaSkin: hs.hotkey callback: /Users/jorgematos/.hammerspoon/keyboard/windows.lua:261: attempt to call a nil value (field '?')
stack traceback:
	/Users/jorgematos/.hammerspoon/keyboard/windows.lua:261: in upvalue 'fn'
	/Users/jorgematos/.hammerspoon/keyboard/windows.lua:217: in function </Users/jorgematos/.hammerspoon/keyboard/windows.lua:215>
--- ERROR ---

**Comment by cmsj** [comment]
Is your Hammerspoon config available to look at somewhere? without seeing what `keyboard/windows.lua` does, it's a bit hard to tell where the problem is 🙂

**Comment by jmatos** [comment]
This is my windows.lua file.

[windows.lua.txt](https://github.com/Hammerspoon/hammerspoon/files/6362040/windows.lua.txt)

**Comment by cmsj** [comment]
Looking at line 261, it's calling a function:

```
fw[winFunction](fw)
```

and if I do this obviously wrong thing in the Hammerspoon Console, I get the same error:

```
> foo = hs.window.focusedWindow()
> foo["?"]()
[string "return foo["?"]()"]:1: attempt to call a nil value (field '?')
```

It's hard to know what's actually being intended here because the actual source of that `?` seems to be a config file called `window-bindings` (per line 221 of your `windows.lua.txt` file).

**Comment by cmsj** [comment]
This appears to be a duplicate of https://github.com/Hammerspoon/hammerspoon/issues/2517

**Comment by jmatos** [comment]
Agreed - seems like a duplicate -  I don't have an interest in rewriting the code that is in my windows.lua - since I didn't create it in the first place and I'm not great at lua - it seems i'll have to keep using the same version of hammerspoon for now.

**Comment by cmsj** [comment]
It may be that the original author has an updated version available for you, or the person in #2517 who fixed theirs.
